cmd/collector/collector_test.go

```diff
@@ -10,12 +10,40 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// TODO
-func TestMetricsUp() {
-	// testutil.ScrapeAndCompare()
-	// "http://localhost:2112/metrics"
-	// # HELP perfpod_memory_collector_up_metric Test metric to confirm skeleton application functionality.
-	// # TYPE perfpod_memory_collector_up_metric gauge
-	// perfpod_memory_collector_up_metric 1
-}
 
+func TestMetricsUp(t *testing.T) {
+
+	// Initialize module/server for standalone integration test
+	go main()
+	time.Sleep(1 * time.Second)
```

Comment:
Nitpick.. 

Does require.Eventuallyf() retry in case the server has not initialized yet? If so can rely on that mechanism and remove this sleep (and if the server takes less than 100ms to initialize, the test would take ~100ms)

Reply:
👍 If you submit I'll review, up to you.